plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -191,33 +255,17 @@ loop:
 		case apiequality.Semantic.DeepEqual(pod, podCopy):
 			// if it validated without mutating anything, use this result
 			allowedPod = podCopy
-			allowingProvider = provider
+			allowingPSP = provider.GetPSPName()
 			break loop
```

Comment:
is the loop label still useful now that we aren't doing anything else in this function?

Reply:
Good point. Fixed.